lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -313,6 +313,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
         'webpacker:compile'
       ].join(',').freeze
 
+      # rubocop:disable Metrics/CollectionLiteralLength
```

Comment:
The `NewRelic::Agent::Configuration::DEFAULTS` hash now exceeds 250 lines, which violates the `Metrics/CollectionLiteralLength` rule. Moving `DEFAULTS` into a separate data storage file, like YAML or JSON would impact many other parts of our system, such as the generation of our documentation using HEREDOCs.

Reply:
^-- This is a summary of a discussion with @fallwith and @hannahramadan about temporarily disabling this rule. Should it be included in a source code comment?

Reply:
It seems fine to me, but i'm good with it either way!

Reply:
Reminder for @hannahramadan to https://github.com/newrelic/newrelic-ruby-agent/pull/2367/commits/6181fe1a91f742080b8513e21632d20456ebaf09 to match